sum_of_multiples.rb

```diff
@@ -0,0 +1,27 @@
+class SumOfMultiples
+
+  def initialize(*num)
+    @array_of_numbers = num
+  end
+
+  def to(multiple_limit)
+    @result_array = []
+    sum_of_multiples=0
+    (0..@array_of_numbers.length-1).each do |i|
```

Comment:
@HardikPurohit If you need the index as well in the loop you can use `each_with_index`